src/components/SendAskMoney.vue

```diff
@@ -92,7 +92,8 @@
           >
             <p class="control has-icons-left custom-search-bar">
               <input
-                v-model="searchName"
+                :value="searchName"
+                @input="(e) => (searchName = e.target.value)"
```

Comment:
`v-on:input` and `@input` are the same, `@input` is just a shorthand syntax.

You could refactor  this and either set what you put in `@input` in `v-on:input`, or the reverse.
I think `@input` is a good choice...